JamGame/Assets/Scripts/Level/Finances/Model.cs

```diff
@@ -5,6 +5,11 @@
 
 namespace Level.Finances
 {
+    public struct Money
+    {
+        public float Value;
+    }
+
     public struct MoneyEarned
     {
         public float Value;
```

Comment:
Why you duplicate data struct? Can you use one in two places?

Reply:
They're semantically different